README.md

```diff
@@ -176,8 +176,6 @@ We're offering support for Chucker on the [#chucker](https://kotlinlang.slack.co
 
 Short `TODO` List for new contributors:
 
-- [ ] Kotlinize classes inside the `.internal` package.
-- [x] Have a empty state graphics/message for requests with no headers.
 - [ ] Increment the test coverage.
```

Comment:
Maybe we can add a couple? Or link to:
https://github.com/ChuckerTeam/chucker/labels/help%20wanted